packages/blade/src/components/Box/__tests__/__snapshots__/Box.web.test.tsx.snap

```diff
@@ -0,0 +1,36 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Box /> should render Box component with the correct styles 1`] = `
+.c0 {
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-align-items: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  -webkit-box-pack: center;
+  -webkit-justify-content: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  -webkit-align-self: center;
+  -ms-flex-item-align: center;
+  align-self: center;
+  padding-top: 8px;
+  padding-bottom: 12px;
+  padding-left: 16px;
+  padding-right: 20px;
+  min-height: 48px;
+  min-width: 48px;
+  max-height: 56px;
+  max-width: 56px;
+}
+
+<div>
+  <div
+    class="c0"
+    display="flex"
```

Comment:
The display probably should not get leaked into the dom.

Reply:
Just doesn't feel right to pollute the html elements, in certain cases I believe react also warns about these if they overlap with any native props. 

It's just weird why styled-components work like this, They have transient props but they [look super ugly](https://styled-components.com/docs/api#transient-props)

Reply:
> Just doesn't feel right to pollute the html elements

Agreed. I'm guessing we'll face this issue as we add more things to props and they start showing up in the DOM 😢

Reply:
Transient props do look ugly. 

We could do this
```
import type { ReactElement } from 'react';
import styled from 'styled-components';
import type { BoxProps } from './Box.d';
import getBoxStyles from './getBoxStyles';

const StyledBox = styled.div<Omit<BoxProps, 'display'> & { $display: BoxProps['display'] }>(
  ({ $display, ...props }) => ({
    ...getBoxStyles({ ...props, display: $display }),
  }),
);

const Box = ({ display, ...props }: BoxProps): ReactElement => {
  return <StyledBox $display={display} {...props} />;
};

export default Box;
```

But I'm not sure if it's worth it. Lets move to doing this if it causes any issues?

Reply:
Yeah for now let's skip it we can relook it later. I've seen some solutions in stackoverflow that destructuring props while accepting them also works.

Reply:
Don't see any sort of warnings in the console 🤔

Reply:
Does it render `flex` property on the div in DOM?

Reply:
It has display as flex but doesn't show in the DOM
<img width="501" alt="Screenshot 2022-07-21 at 5 51 06 PM" src="https://user-images.githubusercontent.com/24487274/180212209-ecf45eb7-4dd4-4dd5-8446-f18f73ad786f.png">